spec/unit/repository_spec.rb

```diff
@@ -5,34 +5,34 @@
   #   stub_command('apt-key list | grep 548C16BF').and_return(true)
   # end
 
-  context 'Ubuntu 12.04' do
-    let(:chef_run) { ChefSpec::SoloRunner.new(:platform => 'ubuntu', :version => '12.04').converge(described_recipe) }
+  context 'Ubuntu 14.04' do
+    let(:chef_run) { ChefSpec::SoloRunner.new(:platform => 'ubuntu', :version => '14.04').converge(described_recipe) }
 
     it 'installs the New Relic apt repository' do
       expect(chef_run).to add_apt_repository('newrelic').with(
         :uri          => 'http://download.newrelic.com/debian/',
         :distribution => 'newrelic',
         :components   => ['non-free'],
-        :key          => 'http://download.newrelic.com/548C16BF.gpg'
```

Comment:
This should have been addressed in the latest version - any change you could rebase?
Happy for the other changes to go in ASAP!

Thanks in advance!
David